assets/sass/components/news.scss

```diff
@@ -0,0 +1,29 @@
+/* =========================================================================
+   News
+   ========================================================================= */
+
+// .article-teaser {}
```

Comment:
Could delete this?

Reply:
Could, but I kind of like an explicit comment saying what the parent class  is if it doesn't have any styles attached.